pkg/customizations/oscap/stage_options.go

```diff
@@ -64,3 +64,38 @@ func CreateTailoringStageOptions(oscapConfig *blueprint.OpenSCAPCustomization, d
 		},
 	)
 }
+
+func CreateRemediationStageOptions(
+	oscapConfig *blueprint.OpenSCAPCustomization,
```

Comment:
It would be better if this function would accept some internal abstraction of the OSCAP config, instead of BP customization.